core/scripts/cli/sponsor/showMarketDetails.js

```diff
@@ -28,56 +24,32 @@ const showMarketDetails = async (web3, artifacts, emp) => {
   const printSponsorSummary = async sponsorAddress => {
     console.group("Summary of your position:");
 
+    const collateral = (await emp.getCollateral(sponsorAddress)).toString();
+
     if (collateral !== "0") {
       const position = await emp.positions(sponsorAddress);
       const isWeth = await getIsWeth(web3, artifacts, collateralCurrency);
       const collateralSymbol = await getCurrencySymbol(web3, artifacts, collateralCurrency);
-
-      // TODO: potentially generalize this for use elsewhere in the CLI tool.
-      const getCollateralizationRatio = async () => {
-        let priceFeed;
-        try {
-          priceFeed = await createReferencePriceFeedForEmp(
-            winston.createLogger({ silent: true }),
-            web3,
-            new Networker(),
-            () => Math.floor(Date.now() / 1000),
-            emp.address
-          );
-        } catch (error) {
-          // Ignore error
-        }
-
-        if (!priceFeed) {
-          return "Unknown";
-        }
-
-        await priceFeed.update();
-        const currentPrice = priceFeed.getCurrentPrice();
-
-        if (!currentPrice) {
-          return "Unknown";
-        }
-
-        const collateralizationRatio = await computeCollateralizationRatio(
-          web3,
-          currentPrice,
-          toBN(collateral.toString()),
-          toBN(position.tokensOutstanding.toString())
-        );
-        const format = createFormatFunction(web3, 2, 4, false);
-        return format(collateralizationRatio.muln(100)) + "%";
-      };
+      const collateralRequirement = await emp.collateralRequirement();
+      const format = createFormatFunction(web3, 2, 4, false);
```

Comment:
You need the last param `false` in this function call. the interface for `createFormatFunction` is:
```
const createFormatFunction = (web3, numDisplayedDecimals, minDisplayedPrecision)
```

The false, I'm assuming, was for rounding `formatWithMaxDecimals` function but this is set to false by default [here](https://github.com/UMAprotocol/protocol/blob/5b47a4ea9a8e62f39598f039150ba49a87212b00/common/FormattingUtils.js#L73)